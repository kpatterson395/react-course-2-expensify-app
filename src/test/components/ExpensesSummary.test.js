import React from 'react';
import { shallow } from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary';

test('should correctly render expenses summary with one expense', () => {
	const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={235} />)
	expect(wrapper).toMatchSnapshot()
})

test('should correctly render expenses summary with multiple expenses', () => {
	const wrapper = shallow(<ExpensesSummary expensesCount={2} expensesTotal={500} />)
	expect(wrapper).toMatchSnapshot()
})