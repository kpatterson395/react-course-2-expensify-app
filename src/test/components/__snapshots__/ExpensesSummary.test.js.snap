// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render expenses summary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totalling in 
    $5.00
  </h1>
</div>
`;

exports[`should correctly render expenses summary with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling in 
    $2.35
  </h1>
</div>
`;
